fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/expressions/Field.java

```diff
@@ -64,6 +64,17 @@ public OneOfThem oneOfThem() {
         return new OneOfThem(fieldName);
```

Comment:
The docs on this method should probably clarify what the default `emptyIsUnknown` behavior is.